VisualStudio/InstallerService.cs

```diff
@@ -72,26 +52,46 @@ async Task RunAsync(string command, Channel? channel, Sku? sku, IEnumerable<stri
             process.WaitForExit();
         }
 
-        async Task<string> DownloadAsync(string bootstrapperUrl, TextWriter output)
+        private string MapChannel(Channel? channel)
```

Comment:
how about using a lambda directly here? (just `=> channel switch ...`)  Also, we don't add the `private` keyword to private methods since it's the default already.